cdi/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~  Copyright 2017 Red Hat, Inc.
+ ~
+ ~  Licensed under the Apache License, Version 2.0 (the "License");
+ ~  you may not use this file except in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing, software
+ ~  distributed under the License is distributed on an "AS IS" BASIS,
+ ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~  See the License for the specific language governing permissions and
+ ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.smallrye.config</groupId>
+    <artifactId>smallrye-config-parent</artifactId>
+    <version>1.7.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>smallrye-config</artifactId>
```

Comment:
shouldn't it be "smallrye-config-cdi" for instance?

Reply:
Ah ok, so you basically assume most of the users out there are using CDI anyway?! I would find it cleaner if CDI enabling would be a separate dependency. I think its like that for most libraries?! Cheers!

Reply:
Well not exactly. We don't really have an idea on who is using or not since everything is bundled together. 

The idea here was to keep the same artifact name for the CDI dependency (which will depend on the standalone implementation, with a new artifact name), so it won't break and if you want to use the non-CDI dependency, you just change it to the new one.

So right now we have:

artifact `smallrye-config` (both CDI / Implementation)

Move to:

artifact `smallrye-core` (Implementation Only)
artifact `smallrye-config` (CDI only code -> depends on smallrye-core)

So this won't break anyone and if you want to use the implementation only without CDI you change the dependency from `smallrye-config` to `smallrye-core`. Does that make sense?

Reply:
@svenhaag let me know if this is ok with you.

Reply:
Hi @radcortez you are awesome! That's just fine. Thank you very much.